google-cloud-clients/google-cloud-core/src/main/java/com/google/cloud/Date.java

```diff
@@ -69,6 +70,38 @@ public static Date parseDate(String date) {
     return new Date(year, month, dayOfMonth);
   }
 
+  /**
+   * Set the Google Date
+   *
+   * @param date the date of the Google Date.
+   * @return java.util.Date
+   */
+  public static java.util.Date toDate(Date date) {
+    Calendar cal = Calendar.getInstance();
+    cal.set(Calendar.HOUR_OF_DAY, 0);
+    cal.set(Calendar.MINUTE, 0);
+    cal.set(Calendar.SECOND, 0);
+    cal.set(Calendar.MILLISECOND, 0);
+    cal.set(date.year, date.month - 1, date.dayOfMonth);
+    return cal.getTime();
+  }
+
+  /**
+   * Set the Java Util Date.
+   *
+   * @param date the date of the java.util.Date
+   * @return Google Java Date
+   */
+  public static Date fromDate(java.util.Date date) {
+    Calendar cal = Calendar.getInstance();
+    cal.setTime(date);
+    cal.set(Calendar.HOUR_OF_DAY, 0);
+    cal.set(Calendar.MINUTE, 0);
+    cal.set(Calendar.SECOND, 0);
+    cal.set(Calendar.MILLISECOND, 0);
+    return new Date(cal.get(Calendar.YEAR), cal.get(Calendar.MONTH) + 1, cal.get(Calendar.DAY_OF_MONTH));
```

Comment:
Add a comment here that calender month starts from 0 while G C J date starts from 1